From a0be377d5346d0fd8b27f57dcf1e28103b257fb4 Mon Sep 17 00:00:00 2001
From: Richard Purdie <richard.purdie@linuxfoundation.org>
Date: Mon, 13 Nov 2023 05:15:58 +0000
Subject: [PATCH] Injection a year based on the current date isn't
 reproducible. Hack this to a specific year for now for reproducibilty and to
 avoid autobuilder failures.

The correct fix would be to use SOURCE_DATE_EPOCH from the environment and
then this could be submitted upstream, sadly my ruby isn't up to that.

Upstream-Status: Pending [could be reworked]
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

---
 Source/JavaScriptCore/generator/GeneratedFile.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/JavaScriptCore/generator/GeneratedFile.rb b/Source/JavaScriptCore/generator/GeneratedFile.rb
index 6ed2b6e4..86a28286 100644
--- a/Source/JavaScriptCore/generator/GeneratedFile.rb
+++ b/Source/JavaScriptCore/generator/GeneratedFile.rb
@@ -25,7 +25,7 @@ require 'date'
 require 'digest'
 
 $LICENSE = <<-EOF
-Copyright (C) #{Date.today.year} Apple Inc. All rights reserved.
+Copyright (C) 2021 Apple Inc. All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
